spec/services/idv/steps/send_link_step_spec.rb

```diff
@@ -68,6 +68,16 @@
       and_return(irs_attempts_api_tracker)
   end
 
+  describe '#extra_view_variables' do
+    it 'includes form' do
+      expect(step.extra_view_variables).to include(
+        {
+          idv_phone_form: be_an_instance_of(Idv::PhoneForm),
+        },
```

Comment:
Technically the curly braces could be optional, though I don't have a strong preference in case you think they help readability.

```suggestion
        idv_phone_form: be_an_instance_of(Idv::PhoneForm),
```